src/core/types/Flux.ts

```diff
@@ -0,0 +1,32 @@
+export interface Signal {
```

Comment:
I'm not a fan of the word ```Signal``` even though we used it a lot when discussing this signal/intent stuff. I think ```Event``` would be way easier to grasp for newcomers to the project, especially since they are generated by the ```EventManager```, but it's an already existing type.
Basically, we have ```DOMEvent ~> EventNormalizer => NormalizedEvent ~> EventManager => Intent ~> Dispatcher```. I don't like ```NormalizedEvent``` either since the fact that they are normalized is just an implementation detail. Couldn't the ```EventNormalizer``` use the same api as the DOMEvent ?